src/data/SpreedSignaling.ts

```diff
@@ -0,0 +1,170 @@
+// NOTE(andrews): This file contains most of the messages that
```

Comment:
I'm not sure, I'm still brainstorming the abstraction. This file is only meant to add type information to Spreed-specific signaling events.